python_scripts/functions.py

```diff
@@ -29,14 +29,11 @@ def load_and_validate_config():
         ext = "json"
     else:
         raise FileNotFoundError(
-            "Expecting a file with the name config.json or config.yaml in working dir."
+            "Expecting a file with the name config.json, config.yaml config.yml) in working dir."
         )
 
     with open(f"config.{ext}", "r") as f:
```

Comment:
should we be assuming that `config.yaml` is in the working directory? Would it be better to make it a parameter?